Content.Client/Weapons/Ranged/Components/MagVisualizer.cs

```diff
@@ -0,0 +1,107 @@
+using Content.Shared.Rounding;
+using Content.Shared.Weapons.Ranged.Systems;
+using JetBrains.Annotations;
+using Robust.Client.GameObjects;
+using SharedGunSystem = Content.Shared.Weapons.Ranged.Systems.SharedGunSystem;
+
+namespace Content.Client.Weapons.Ranged.Components;
+
+[UsedImplicitly]
+public sealed class MagVisualizer : AppearanceVisualizer
+{
```

Comment:
this aint a visualizer system (unless this is just ported/moved and diff is lying)

Reply:
It's just moved (the diff fell over).